sky/provision/kubernetes/instance.py

```diff
@@ -867,6 +872,32 @@ def _create_resource_thread(i: int):
                 existing_rules)
             pod_spec_copy['spec']['affinity'] = pod_spec_config
 
+        # Create PVCs for the pod
+        pvcs_copy = copy.deepcopy(pvcs)
+        for pvc in pvcs_copy:
+            # Remove auto_delete from the pvc spec
+            pvc.pop('autoDelete', None)
+            path = pvc.pop('path', None)
+            original_name = pvc['metadata']['name']
+            new_name = f'{original_name}-{i}'
+            pvc['metadata']['name'] = new_name
```

Comment:
How does this work with when `--num-nodes 2`? IIUC, this will create two PVCs and attach them to the the nodes instead of using the same one across nodes?